cmd/legacy/today/today.go

```diff
@@ -59,18 +58,14 @@ func Summary(v *viper.Viper) (string, error) {
 		return "", fmt.Errorf("failed to initialize api client: %w", err)
 	}
 
-	now := time.Now()
-	todayStart := time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, now.Location())
-	todayEnd := time.Date(now.Year(), now.Month(), now.Day(), 23, 59, 59, int(time.Second-time.Nanosecond), now.Location())
-
-	summaries, err := apiClient.Summaries(todayStart, todayEnd)
+	parsed, err := apiClient.Summary()
```

Comment:
it seems that `summary` or `today` makes more sense here.
```suggestion
	summary, err := apiClient.Summary()
```

Reply:
`summary` conflicts with a module name, so I'll use `today`.